core/src/air/public_values.rs

```diff
@@ -0,0 +1,103 @@
+use core::fmt::Debug;
+
+use std::{
+    array::IntoIter,
+    ops::{Index, IndexMut},
+};
+
+use p3_field::Field;
+use serde::{Deserialize, Serialize};
+
+use super::Word;
+
+// TODO:  Create a config struct that will store the num_words setting and the hash function
+//        and initial entropy used.
+const PV_DIGEST_NUM_WORDS: usize = 8;
+
+/// The PublicValuesDigest struct is used to represent the public input digest.  This is the hash of all the
+/// bytes that the guest program has written to public input.
+#[derive(Serialize, Deserialize, Clone, Copy, Default, Debug)]
+pub struct PublicValuesDigest<T>([T; PV_DIGEST_NUM_WORDS]);
+
+/// Convertion from a byte array into a PiDigest<u32>.
```

Comment:
nit: Convertion -> Conversion + this file still mentions PiDigest / public inputs, should be changed to PublicValuesDigest